src/com/georgidinov/domain/record/BorrowRecord.java

```diff
@@ -0,0 +1,103 @@
+package com.georgidinov.domain.record;
+
+import com.georgidinov.domain.book.Book;
+import com.georgidinov.domain.person.Person;
+
+import java.time.LocalDate;
+
+import static com.georgidinov.util.MyDateFormatter.getDateString;
+import static com.georgidinov.util.MyPunctuation.COMA;
+import static com.georgidinov.util.MyPunctuation.SPACE;
+
+public class BorrowRecord {
+
+    private static final String POSTPONE_REQUEST = "Postpone Request";
+    private static final String GRANTED = "Granted";
+    private static final String DENIED = "Denied";
+    public static final String NEW = "New";
+    public static final String DUE_DATE = "Due Date";
+    public static final String IS = "Is";
```

Comment:
Why we don't use whole sentences for the messages, instead of building them word by word